WebWasted/Controllers/FoodsController.cs

```diff
@@ -30,11 +30,29 @@ public IEnumerable<Food> Get()
                 return dataContext.Foods.ToList();
             }
         }
- 
+
+        [HttpGet("{id}")]      //e.g. https://localhost:5000/api/foods/3
```

Comment:
idk if it's the best choice to use this id as a part of link to send the needed info but I guess for now this will work.

It's not good because anyone could access your cart, they would only need to write a number at the end